q2_types/feature_data/_format.py

```diff
@@ -185,10 +185,19 @@ def sniff(self):
     AlignedDNAFASTAFormat)
 
 
+class DifferentialFormat(model.TextFileFormat):
+    def validate(*args):
```

Comment:
I think it would be really important to actually implement some format validation. Do you need a hand with brainstorming ideas for that?

Reply:
There are a couple of low hanging fruits for validation, namely

1. Test to see if all of the entries are floating point
2. Make sure that there are no infs or nans

Reply:
These sound good to me.